processor/cumulativetodeltaprocessor/config.go

```diff
@@ -24,15 +24,19 @@ import (
 type Config struct {
 	config.ProcessorSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
-	// List of cumulative sum metrics to convert to delta
+	// List of cumulative metrics to convert to delta. Default: converts all cumulative metrics to delta.
 	Metrics []string `mapstructure:"metrics"`
+
+	// The total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely.
```

Comment:
```suggestion
	// MaxStale is the total time a state entry will live past the time it was last seen. Set to 0 to retain state indefinitely.
```
style nit: Go style convention is that exported field comments should start with field name.